app/models/user.rb

```diff
@@ -23,6 +23,7 @@ class User < ActiveRecord::Base
 
   before_create :set_token
   validates :time_format, inclusion: { in: TIME_FORMATS }
+  validates :external_id, presence: :true, unless: :integration?
```

Comment:
If we add a validation like this, we should have a plan on what to do with users that fail the validation.  Not okay to have a bunch of saved records in the database that aren't considered valid anymore.